src/sth_database.js

```diff
@@ -0,0 +1,128 @@
+/* globals module, process, require */
+
+(function() {
+  "use strict";
+
+  var mongoose = require('mongoose');
+
+  var sthConfig, sthHelper, connectionURL, eventSchema, aggregatedSchema;
+
+  function connect(authentication, host, port, database, callback) {
```

Comment:
JSDoc comments included! ;) Sorry about that, I forgot to do it from the beginning ;) I'll mention you as soon as I pushed the changes to the repo ;) Thanks!